app/src/modules/iot_pollution/iot_pollution.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * This is your main app file. Please refer to the documentation for more information.
+ */
+
+/**
+ * If you need to import extra node_modules, use "npm install xxx --save" and place them there.
+ */
+
+var Promise = require("bluebird");
+var _ = require('@qwant/front-i18n')._;
+var moment = require('moment');
+var config = require('@qwant/config');
+config.import('iot_pollution');
+Object.keys(config).forEach(function(elem) {
+    config_set(elem, config[elem]);
+});
+
+module.exports = {
+
+    /**
+     * (NEEDED)
+     *	This function uses 3 parameters :
+     *	- values : This is an array of values caught by regex.
+     *	For example, if you use the keyword "test" with the trigger "start", and you type "test working?",
+     *  values would be like this :
+     *  	* values[0] = "test working?"
+     *  	* values[1] = "test"
+     *  	* values[2] = "working?"
+     *  But, if you use the trigger "strict", there will be only one value in this array (values[0] = "test working?")
+     *  - proxyURL : If you need to call an external API, use the package "request" with proxyURL as value for
+     *  "proxy" attribute (you can refer to weather IA to check how to use it properly)
+     *  - language : Current language called
+     * @returns data to be displayed.
+     */
+
+    getData: function (values, proxyURL, language, i18n) {
+      const _ = i18n._;
+      return new Promise(function (resolve, reject) {
+        if (values && values[0]) {
+          // when generating our IA, we chose a "strict" trigger with "bikes" as the keyword.
+          // It means it will trigger the IA if the query is strictly equal to "bikes", and nothing else.
+          // So when using a "strict" trigger, you don't need to check values[2] as it doesn't exist.
+          const requestUser = values[2]
+          const apiAddress = config_get('iot_pollution.api').iotAPI
+          // Declare redisTools
+          const CACHE_KEY = 'iot-pollution-cache';
+          const CACHE_EXPIRE = 7200;
+          const redisTools = require('../../redis_tools');
+          redisTools.initRedis();
+          console.log(requestUser);
+          // Checking the cache to avoid requesting the API
+
+          redisTools.getFromCache(CACHE_KEY).then((cached) => {
```

Comment:
Delete the call to the redis cache. There is already a cache that is used with the `cache` parameter.

Reply:
Do I completly remove redisTools ?